### ISSUE TITLE
[feature] dns over tls

### ISSUE THREAD

**Issue by cubakcabak** [SEVENNetworks/AdClear#60]
since the listed public DNS provider support dns-over-tls. it would be added extra layer of security

**Comment by csandlin1** [comment]
This is something that we have on our backlog of things to add, but it is low priority at the moment.

**Comment by cubakcabak** [comment]
@csandlin1 actually it could be proven to be very useful very much.

imagine this scenario, user couldn't access certain website/service, not because the website is ads ridden, but because it being intercepted/redirected/blocked by their internet provider using the oldest trick in the book of networking, an old general dns protocol.

now, that's whats happened to several country who's by their government "policy" or business "decisions" decided to do those practice.

people need to to use "real vpn" just to access information on website/service that those government/business deem "inappropriate" to them.

government case: reddit because of porn, disregarding the usefulness of other information provided in the website. whatsapp/telegram because they cant have backdoor to the service.

provider case: netflix, because it compete with their provided service.

and it's real, because i live in a country that i described.

so, adclear will be definitely helping not just blocking ads, but keep the freedom of the internet.

anyway, thank you all the developer of adclear for making this great software available. it's helped so much so far in making my mediocre not so high-end phone became useful again.

**Comment by csandlin1** [comment]
I totally understand why DNS over TLS is so important. The only reason it is further back on our list is due to integrating things like HTTP/2, which a rapid majority of sites have been transitioning to. For privacy features in the meantime, you can use the Proxy setting in AdClear if you use Android 6 or above, but I know that doesn't exactly fit your needs. I'll be sure to bring this up again at our next meeting. Thank you for your suggestion.

**Comment by DustyWMorgan** [comment]
Let's make sure stale doesn't close this feature request. We are still considering support for this if the feature would be used widely enough.
